src/content/advanced/cluster-management/new-cluster-api/index.md

```diff
@@ -0,0 +1,124 @@
+
+## Motivations for Transitioning to CAPI
+
+ * **Community-Driven Development**: CAPI is a widely adopted and actively developed project within the Kubernetes community. By embracing CAPI, we benefit from the collective expertise of many contributors, ensuring the continued advancement and reliability of our cluster management solution.
+ * **Vendor-Neutrality and Future-Proofing**: CAPI adheres to a neutral and open API, shielding our customers from vendor lock-in and ensuring their ability to seamlessly integrate with the evolving cloud landscape. This open approach also facilitates faster innovation and integration with new technologies.
+ * **Greater Flexibility and Customization**: CAPI provides a more expressive and flexible API for defining and managing Kubernetes clusters. With CAPI, we can cater to diverse customer requirements, enabling them to deploy into existing infrastructure and customize cluster configurations to suit their specific needs.
+ * **Expanded Tooling Ecosystem**: CAPI is compatible with a vast array of tools and solutions developed by the community. By adopting CAPI, we gain access to a rich toolkit for cluster management, automation, and orchestration.
+
+## Benefits of CAPI
+
+* **Enhanced Flexibility**: CAPI empowers us to provide our customers with greater control over their infrastructure, enabling them to deploy into existing networks, customize cluster configurations, and integrate with a wider range of infrastructure providers.
+* **Future-Proofed API**: CAPI's open and vendor-neutral API ensures our platform's viability in the long term, allowing us to adapt to evolving cloud architectures and technologies without being constrained by proprietary solutions.
+* **Broad Tooling Compatibility**: Leveraging CAPI allows us to leverage a multitude of tools developed by the community, enhancing our ability to automate, manage, and orchestrate Kubernetes clusters effectively.
+* **Support for Managed Control Planes**: CAPI enables us to seamlessly integrate with managed control planes like EKS and AKS, providing customers with a choice of deployment options.
+* **Simplified Provider Addition**: CAPI's composable architecture makes it easier to add new infrastructure providers, expanding our offerings beyond AWS, Azure, and our current on-premises solutions.
+* **Multi-Cloud Capabilities**: CAPI supports managing workload clusters from multiple providers under a single management cluster, enabling true multi-cloud deployment scenarios.
+
+## Differences between CAPI and Legacy Cluster Management
```

Comment:
```suggestion
## Differences between CAPI and Vintage Cluster Management
```
We always refer to our current product as Vintage

Reply:
Imo the word "vintage" is confusing in the docs. So I prefer to use a more common word for this.